diff --git a/Makefile.linux b/Makefile.linux
index 7b27008..bc7c4f1 100644
--- a/Makefile.linux
+++ b/Makefile.linux
@@ -20,8 +20,10 @@ COMMANDS += containerd-shim containerd-shim-runc-v1 containerd-shim-runc-v2
 
 # check GOOS for cross compile builds
 ifeq ($(GOOS),linux)
+ifneq ($(GOARCH),ppc64)
 	GO_GCFLAGS += -buildmode=pie
 endif
+endif
 
 # amd64 supports go test -race
 ifeq ($(GOARCH),amd64)
